core/google/cloud/_helpers.py

```diff
@@ -28,10 +28,16 @@
 
 import google.auth
 from google.protobuf import timestamp_pb2
-try:
+import google_auth_httplib2
+
+try:  # pragma: NO COVER
```

Comment:
grpc isn't installed. The unit tests actually hit the other branch. I don't know why this worked before.